signac/contrib/project.py

```diff
@@ -506,6 +515,8 @@ def detect_schema(self, exclude_const=False, subset=None, index=None):
         statepoint_index = self.build_job_statepoint_index(exclude_const=exclude_const, index=index)
         return ProjectSchema.detect(statepoint_index)
 
+    @deprecated(deprecated_in="1.3", removed_in="2.0", current_version=__version__,
+                details="Use find_jobs().ids instead.")
     def find_job_ids(self, filter=None, doc_filter=None, index=None):
```

Comment:
I didn't realize this until now -- we have a `find_job_ids` function in both `JobSearchIndex` and `Project`. We're deprecating both, I think. Please apply the same changes as above (make a private method with the implementation, and a deprecated public method with the same arguments that calls the private method).

Reply:
Good catch, I knew the method was in both places but misread where the private method was added. Definitely need to have a private method in the `Project` namespace for the internal calls. That's even more important than the deprecation in `JobSearchIndex` I think.

Reply:
> I didn't realize this until now -- we have a `find_job_ids` function in both `JobSearchIndex` and `Project`. We're deprecating both, I think. Please apply the same changes in `Project` that you did to `JobSearchIndex` (make a private method with the implementation, and a deprecated public method with the same arguments that calls the private method).

I missed that earlier. No need to deprecate methods, just the one in the `Project` class. The `JobSearchIndex` is only implicitly exposed to users as a return value of a now deprecated method. See also my latest review.